rewrite-java-test/src/test/java/org/openrewrite/java/RemoveObjectsIsNullTest.java

```diff
@@ -125,27 +101,26 @@ public void test() {
 
     @Issue("https://github.com/openrewrite/rewrite/issues/1547")
     @Test
-    void transformCallToNonNullNeedsParentheses() {
```

Comment:
Was this test completely removed by intention?

Reply:
Yes after the recipe changes this was a duplicate. We can revise this of course based on the other feedback here, but up to this commit it was intentional.